console/console-init/ui/src/graphql-module/queries/iot_device.ts

```diff
@@ -117,12 +117,18 @@ const SORT_RETURN_ALL_DEVICES_FOR_IOT_PROJECT = (sortBy?: ISortBy) => {
       case 1:
         orderBy = "`$.deviceId` ";
         break;
-      case 2:
```

Comment:
what will happen for case 2?

Reply:
If we are not having sorting for that column. I would suggest to keep the case i.e `case 2: break;`

Reply:
Done.